content/docs/release-notes/archive/21-11-release-notes.md

```diff
@@ -332,6 +332,28 @@ Open 3D Engine Project Manager now has improvements which allow the support of G
 
   To work around this issue, run AssetProcessor to process project assets before launching Editor for the first time. After all assets are processed initially, Editor can be launched without manually running AssetProcessor.
 
+* If the user has previously used O3DE and an `o3de_manifest.json` file exists in their `/Users/(username)/.o3de` directory, when they go to build a project using the current installer version O3DE, it may fail with an error similar to the following:
```

Comment:
```suggestion
* If the user has previously used O3DE and an `o3de_manifest.json` file exists in their `/Users/(username)/.o3de` directory, when they go to build a project using the current installer version of O3DE, it may fail with an error similar to the following:
```